dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/MasterServer.java

```diff
@@ -120,7 +124,9 @@ public void run() throws SchedulerException {
         this.eventExecuteService.start();
         this.failoverExecuteThread.start();
 
-        this.schedulerApi.start();
+        if (masterConfig.isSchedulerEnabled()) {
```

Comment:
Every time this config is modified, the master needs to be restarted. I don't quite understand under what circumstances the master's scheduling needs to be turned off, and whether this will cause other potential problems.

Would you please describe more the usage scenarios of this config?

Reply:
> Every time this config is modified, the master needs to be restarted. I don't quite understand under what circumstances the master's scheduling needs to be turned off, and whether this will cause other potential problems.
> 
> Would you please describe more about the usage scenarios of this config?

+1

Reply:
+1,  there is no need to add this config, please provide more info.